package.json

```diff
@@ -100,24 +98,19 @@
     "@ethereumjs/config-prettier": "^1.1.0",
     "@ethereumjs/config-tsc": "^1.1.0",
     "@ethereumjs/config-tslint": "^1.1.0",
+    "@types/mocha": "^5.2.7",
     "@types/node": "^11.9.0",
-    "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babelify": "^8.0.0",
-    "browserify": "^14.0.0",
     "contributor": "^0.1.25",
-    "coveralls": "^3.0.0",
     "husky": "^2.1.0",
-    "istanbul": "^0.4.1",
     "karma": "^4.0.0",
-    "karma-browserify": "^5.0.0",
     "karma-chrome-launcher": "^2.0.0",
-    "karma-detect-browsers": "2.2.6",
-    "karma-env-preprocessor": "^0.1.1",
     "karma-firefox-launcher": "^1.0.0",
     "karma-mocha": "^1.3.0",
+    "karma-typescript": "^4.1.1",
     "mocha": "^6.0.0",
+    "nyc": "^15.0.0",
```

Comment:
For the record: switched from `istanbul` to `nyc`, dropped various `babel` and `karma` related dev dependencies.